include/giskard_core/yaml_parser.hpp

```diff
@@ -1596,6 +1596,49 @@ namespace YAML {
     }
   };
 
+  //
+  // parsing of alias reference specs
+  //
+
+  inline bool is_alias_reference(const Node& node)
+  {
+    if(!node.IsScalar() || is_const_double(node))
```

Comment:
Ah yes, I was wondering what IsScalar() actually tests for. Now the condition makes sense.